relay/sources/rudder-package/src/database.rs

```diff
@@ -221,19 +221,64 @@ impl Database {
             .map(|p| format!("enable {}", p.metadata.name))
             .collect::<Vec<String>>()
             .join("\n");
-        fs::write(backup_path, saved)?;
+        fs::write(backup_path, saved).with_context(|| {
+            format!(
+                "Failed to save the plugins statuses in the backup file {}",
+                backup_path.to_string_lossy()
+            )
+        })?;
+        info!(
+            "Plugins statuses saved in {}",
+            backup_path.to_string_lossy()
+        );
         Ok(())
     }
 
+    fn read_plugin_status_line_from_backup(&self, line: &str, webapp: &mut Webapp) -> Result<()> {
```

Comment:
```suggestion
    fn apply_plugin_status_line_from_backup(&self, line: &str, webapp: &mut Webapp) -> Result<()> {
```

it does more than reading